packages/runtime/src/Runtime.ts

```diff
@@ -258,6 +259,10 @@ export abstract class Runtime<TConfig extends RuntimeConfig = RuntimeConfig> {
             .factory(() => this.getAccountController().tokens)
             .scope(Scope.Request);
 
+        Container.bind(PublicRelationshipTemplateReferencesController)
+            .factory(() => this.getAccountController().publicRelationshipTemplateReferences)
+            .scope(Scope.Singleton);
```

Comment:
Just out of interest: Many Controller have `.scope(Scope.Request)`, some have `.scope(Scope.Singleton)`. What's the difference here?

Reply:
Sigleton is only for anonymous Controllers. So this is wrong and will explode when more than one account exists.